M/MLX_C/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MLX_C"
+version = v"0.0.11" # This is an unofficial version number - used for git commit sha (126253...) matching MLX v0.20.0
+
+sources = [
+    GitSource("https://github.com/ml-explore/mlx-c.git", "126253627b95a0803273f44655fdaf5e7be4fbe1"),
+    ArchiveSource("https://github.com/roblabla/MacOSX-SDKs/releases/download/macosx14.0/MacOSX14.0.sdk.tar.xz",
+                  "4a31565fd2644d1aec23da3829977f83632a20985561a2038e198681e7e7bf49"),
+    DirectorySource("./bundled"),
+]
+
+script = raw"""
+# MLX hack
+cd $includedir
+mv mlx/io.h mlx/io.h.bak
+cp mlx/io.h.bak mlx/io.h
+mv mlx/backend/metal/metal.h mlx/backend/metal/metal.h.bak
+cp mlx/backend/metal/metal.h.bak mlx/backend/metal/metal.h
```

Comment:
What number 2?

Reply:
Same explanation as above.

Reply:
Luckily superfluous now, with MLX v0.21.0 (which include the std::unordered_map patch).